packages/types/src/index.spec.ts

```diff
@@ -8,7 +8,7 @@ import { Constructor } from './types';
 import * as Classes from './index.types';
 
 const Types = Classes as { [index: string]: Constructor };
-const UNCONSTRUCTABLE = ['origin', 'usize'];
+const UNCONSTRUCTABLE = ['origin', 'usize', 'vote'];
```

Comment:
why is this unconstructable?

Reply:
it seems nonsensical to me to construct a Vote with a default value

Reply:
yeah, you're right